free-3.md

```diff
@@ -10,34 +10,41 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Many students in CS70 are frusterated by the makeFiles the use to compile and run their code with all the correct dependencies. They may spend hours debugging trying to find what is wrong with their program, when really its just an issue with the makeFile. I would like to make a tool like sbt that works for C++ instead. This would recognize when a dependent file was updated and saved and then recompile and run. This language would be clearer and simpler to use, allowing students to focus their time and energy on their actual C++ code and make the debugging process faster and easier by allow them to make minor changes to their code, save, and then have everything automatically recompile and run.
 
```

Comment:
Do you think this could be extended beyond CS70 students or are you intending to make it specifically for the class?